app/controllers/concerns/authentication.rb

```diff
@@ -14,7 +14,7 @@ def authenticated?
   def authenticate
     if !authenticated?
       session["post_dfe_sign_in_path"] = request.fullpath
-      redirect_to sign_in_path
+      redirect_to sign_in_path({ support: params[:controller].start_with?("support") })
```

Comment:
Discussed on slack, would be good to only populate this param if we're in dev or qa, as it's non-functional in prod